contracts/factory/src/tests/setup.rs

```diff
@@ -41,7 +39,14 @@ pub fn deploy_factory_contract<'a>(
     let admin = admin.into().unwrap_or(Address::generate(env));
     let factory = FactoryClient::new(env, &env.register_contract(None, Factory {}));
     let multihop_wasm_hash = install_multihop_wasm(env);
-
-    factory.initialize(&admin, &multihop_wasm_hash);
+    let whitelisted_accounts = vec![
+        env,
+        Address::from_string(&String::from_str(
+            env,
+            "CBT4WEAHQ72AYRD7WZFNYE6HGZEIX25754NG37LBLXTTRMWKQNKIUR6O",
+        )),
```

Comment:
What address is this? We need to either require it or pass as an argument.

Reply:
fixed